UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/Home/Index.cshtml

```diff
@@ -1,100 +1,28 @@
-                        Maritime Safety Information
-                    </h1>
-                    <h3>
-                        View the latest safety-critical navigational information
-                    </h3>
-                    <h3>
-                        If you are an ADMIRALTY Distributor, please sign in to see more content
-                    </h3>
-                </div>
-            </div>
-            <div id="home-nav" class="row">
-                <div id="nm-home" class="panel col-md-6">
-                    <a href="@Url.Action("Index", "NoticesToMariners")" class="stretched-link"></a>
-
-                    <div class="title">
-                        <h2>Notices to Mariners (NMs)</h2>
-                    </div>
-
-                    <p class="text">
-                        Weekly updates that provide bridge crews with the latest safety-critical navigational information.
-                        They can be downloaded for free here, or purchased as weekly bulletins from your ADMIRALTY Distributor.
-                    </p>
-
-                    <div class="cta">
-                        <h4>Go to Notices to Mariners</h4>
+<div class="container-fluid">    
+    <div class="row justify-content-lg-center">
+        <div class="col-lg-9">
+            <h1 id="headingLevelOne" class="mt-0">Maritime Safety Information</h1>
+            <p class="intro-text">View the latest safety-critical navigational information </p>
+            <p class="intro-text">If you are an ADMIRALTY Distributor, sign in to see more content </p>
+            <div class="row mb-5">
+                <div class="col-md-6 pe-md-0">
+                    <div class="nm-link">
+                        <h2>Notice to Mariners (NMs)</h2>
+                        <p>  Weekly updates that provide bridge crews with the latest safety-critical navigational information. They can be downloaded for free here, or purchased as weekly bulletins from your ADMIRALTY Distributor.</p>
+                        <a href="@Url.Action("Index", "NoticesToMariners")" class="bottom-link">Go to Notice to Mariners</a>
                     </div>
                 </div>
-                <div id="rnw-home" class="panel col-md-6">
-                    <a href="@Url.Action("Index", "RadioNavigationalWarnings")" class="stretched-link"></a>
-
-                    <div class="title">
-                        <h2>Radio Navigational Warnings (RNWs)</h2>
-                    </div>
-
-                    <p class="text">
-                        NAVAREA I and UK Coastal navigational warnings reproduced online.
-                    </p>
-
-                    <div class="cta">
-                        <h4>Go to Radio Navigational Warnings</h4>
+                <div class="col-md-6 ps-md-0">
+                    <div class="rnw-link">
+                        <h2>Radio Navigational  Warnings (RNWs)</h2>
+                        <p>NAVAREA 1 and UK Coastal navigation warnings reproduced online</p>
```

Comment:
Add dot/punctuation mark at end to make it consistent with NM tab.
```suggestion
                        <p>NAVAREA 1 and UK Coastal navigation warnings reproduced online.</p>
```

Reply:
Changes done.